src/types/profile.js

```diff
@@ -326,5 +335,6 @@ export type ProfileMeta = {|
  */
 export type Profile = {
   meta: ProfileMeta,
+  pages: PageList,
```

Comment:
Same objection here: Either this needs to be `pages?` or you need to increase the profile version and write an upgrader that replaces missing pages arrays with an empty pages array.